cmd/minikube/cmd/config/util.go

```diff
@@ -137,20 +137,28 @@ func EnableOrDisableAddon(name string, val string) error {
 	return enableOrDisableAddonInternal(addon, cmd, data, enable)
 }
 
-func enableOrDisableAddonInternal(addon *assets.Addon, cmd bootstrapper.CommandRunner, data interface{}, enable bool) error {
-	var err error
-	// check addon status before enable/disable it
+func isAddonAlreadySet(addon *assets.Addon, enable bool) error {
+
 	addonStatus, err := addon.IsEnabled()
 
 	if err != nil {
-		exit.WithError("Failed to get the addon status ", err)
+		return errors.Wrap(err, "get the addon status")
 	}
 
 	if addonStatus && enable {
-		console.ErrStyle(console.Conflict, "addon %s was already enabled.", addon.GetAddon())
-		os.Exit(exit.Unavailable)
+		return fmt.Errorf("addon %s was already enabled", addon.GetAddon())
```

Comment:
for more consistency, lets use errors.Wrap in stead of `fmt.Errorf`

Reply:
Thanks Medya, should not errors.Wrap for this case